app/assets/stylesheets/_vocab_sheet.scss

```diff
@@ -173,33 +175,21 @@
     border: 1px solid $light-med-grey;
     margin-right: 11px;
     background: white;
-  
-    .item-name {
```

Comment:
Why did this and then item-maori-name classes get removed? I can see theyre still attached to their inputs in the view file so are their styles just being set somewhere else now?

Reply:
The only styles in those two rules were duplicates that are now inside `.vocab-sheet__text-input` on line 153 above, so there wasn't a need to repeat the rules.